scaling_performance/scaling_cluster_monitoring_operator.adoc

```diff
@@ -0,0 +1,152 @@
+[[scaling-performance-cluster-metrics]]
+= Scaling Cluster Metrics
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+link:https://github.com/openshift/cluster-monitoring-operator[cluster-monitoring-operator]. As an
+{product-title} administrator, you can view system resources, containers and components metrics in
+one dashboard interface.
+
+This topic provides information for scaling the metrics components.
+
+ifdef::openshift-enterprise[]
+[[cluster-metrics-horizontal-pod-autoscaling]]
+[NOTE]
+====
+Autoscaling the metrics components.
+====
+endif::[]
+
+
+[[metrics-recommendations-for-OCP]]
+== Recommendations for {product-title}
+
+* Run metrics pods on dedicated {product-title}
+xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure
+nodes].
+* Use persistent storage when configuring metrics. Set
+`USE_PERSISTENT_STORAGE=true`.
+* use GlusterFS as storage on top of non-volatile memory express (NVMe) drives link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs[*openshift_storage_glusterfs*].
```

Comment:
Use GlusterFS

Reply:
Is *openshift_storage_glusterfs* an example of NVMe drives? If so, maybe we can reword this. As it currently reads, the link to *openshift_storage_glusterfs* looks out of place and I feel like the sentence can end after "(NVMe) drives ".

Reply:
@ekuric can you confirm this please ?!

Reply:
@mrsiano @ahardin-rh I think link should be removed and maybe better to say
- `it is recommended to use NVMe/SSD hardware as block devices for OCS - OpenShift Container Storage which will be providing storage space for Prometheus pods`